### ISSUE TITLE
Apostrophe display bug in sub-title

### ISSUE THREAD

**Issue by RhettTrickett** [able-bio/suggestion-box#25]
Raised by @Minenash

_when going back to edit an article with a ‘ in the subtitle, then in the editor it’s replaced with &#39; which then gets placed as the actual subtitle. And every time I hit edit without fixing it, it duplicates._

**Comment by Minenash** [comment]
Here's an example I ran into today:
***
![Apostrophe](https://imgur.com/hMGK4IB.png)
***

**Comment by RhettTrickett** [comment]
Yeah, I saw this and fixed it for you. I'll make some time to tackle this one over the weekend.

**Comment by Minenash** [comment]
When? I fixed it like 10 times (post publish) over the course of re-editing it xD
